CoffeeChat.xcodeproj/project.pbxproj

```diff
@@ -1045,7 +1045,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = ZGMCXU7X3U;
 				INFOPLIST_FILE = CoffeeChat/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
```

Comment:
do we want 14? I think decided 13 is good

Reply:
It won't let me run the simulator if I use 13. It says "Compiling for iOS 13.0, but module 'FutureNova' has a minimum deployment target of iOS 14.0" ?